js.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+function Person(name, dateOfBirth, amount) {
+    this.name = name;
+    this.dateOfBirth = dateOfBirth.split('.').reverse();
+    this.amount = amount;
+    this.currentAmount = this.amount;
+    let [year, month, day] = this.dateOfBirth;
+    this.accountHistory = [{'Initial': amount}];
+
+    this.operations = function (value, str) {
+        this.accountHistory.push({[str] : value})
+    }
+
+    this.age = function getAge() {
+        let currentYear = new Date().getFullYear();
+        let currentMonth = new Date().getMonth() + 1;
+        let currentDay = new Date().getDate();
+        if (currentMonth < month || currentMonth === month && currentDay < day ) {
+            return (currentYear - year - 1);
+        }
+        return currentYear - year;
```

Comment:
you can create a variable for `currentYear - year` and return variable or variable + 1;